lib/chunks.js

```diff
@@ -494,10 +494,8 @@ module.exports.diffChunks = async ({ coursePath, courseId, courseData, changedFi
       );
       if (
         hasClientFilesCourseInstanceDirectory &&
-        (!existingCourseChunks.courseInstances[ciid] ||
-          !existingCourseChunks.courseInstances[ciid].clientFilesCourseInstance ||
-          (changedCourseChunks.courseInstances[ciid] &&
-            changedCourseChunks.courseInstances[ciid].clientFilesCourseInstance))
+        (!existingCourseChunks.courseInstances[ciid]?.clientFilesCourseInstance ||
+          changedCourseChunks.courseInstances[ciid]?.clientFilesCourseInstance)
```

Comment:
These are just cleanups, right? Not connected to the bug itself?

Reply:
Correct - just cleanup!

Reply:
Thanks!